docker/dev-builder/ubuntu/Dockerfile

```diff
@@ -26,7 +26,12 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
 
 # https://github.com/GreptimeTeam/greptimedb/actions/runs/10935485852/job/30357457188#step:3:7106
 # `aws-lc-sys` require gcc >= 10.3.0 to work, hence alias to use gcc-10
-RUN alias gcc="gcc-10" && alias cc="gcc-10"
+RUN apt-get remove -y gcc-9 g++-9 cpp-9 && \
```

Comment:
Can we alias `g++` and `cpp`?

Reply:
> Can we alias `g++` and `cpp`?

On both my machine and this image `namei /usr/bin/g++` and `namei /usr/bin/cpp` both point to same executable, so I think yes?